gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-subscriptions/subscriptions-details/subscriptions-details.component.ts

```diff
@@ -167,6 +163,15 @@ export class SubscriptionsDetailsComponent implements OnInit {
           }
         }
 
+        if (plan.planMode === 'PUSH' && subscription.consumerConfiguration != null) {
```

Comment:
```suggestion
        if (plan.planMode === 'PUSH' && !!subscription.consumerConfiguration) {
```
WDYT in case consumer configuration is undefined?